src/main.css

```diff
@@ -4,7 +4,7 @@
 
 html {
 	font-family: 'Montserrat', sans-serif;
-	font-size: 15px;
+	font-size: 14px;
```

Comment:
Are there any plans to pull out all these numerical values for things like spacing and font sizes into variables?

Reply:
Good point.